pypeit/images/combineimage.py

```diff
@@ -149,16 +152,24 @@ def run(self, bias=None, flatimages=None, ignore_saturation=False, sigma_clip=Tr
         img_list = [img_stack]
         var_stack = utils.inverse(ivar_stack)
         var_list = [var_stack, rn2img_stack]
-        img_list_out, var_list_out, outmask, nused = combine.weighted_combine(
-            weights, img_list, var_list, (mask_stack == 0),
-            sigma_clip=sigma_clip, sigma_clip_stack=img_stack, sigrej=sigrej, maxiters=maxiters)
+        if combine_method == 'weightmean':
+            img_list_out, var_list_out, gpm, nused = combine.weighted_combine(
+                weights, img_list, var_list, (mask_stack == 0),
+                sigma_clip=sigma_clip, sigma_clip_stack=img_stack, sigrej=sigrej, maxiters=maxiters)
+        elif combine_method == 'median':
+            img_list_out = [np.median(img_stack, axis=0)]
+            var_list_out = [np.median(var_stack, axis=0)]
+            var_list_out += [np.median(rn2img_stack, axis=0)]
+            gpm = np.ones_like(img_list_out[0], dtype='bool')
+        else:
+            msgs.error("Bad choice for combine.  Allowed options are 'median', 'weightmean'.")
 
         # Build the last one
         final_pypeitImage = pypeitimage.PypeItImage(img_list_out[0],
                                                     ivar=utils.inverse(var_list_out[0]),
                                                     bpm=pypeitImage.bpm,
                                                     rn2img=var_list_out[1],
-                                                    crmask=np.invert(outmask),
+                                                    crmask=np.invert(gpm),
```

Comment:
I don't think there is much difference between invert and logical_not, I think they are essentially equivalent and we use invert everywhere in the code. Do you have a reason to prefer logical_not?

Reply:
`logical_not` is safer, in case the type switches from boolean to an integer (it was discussed briefly in PR #918). Here is an example:
```
x = [True, False]
print(np.invert(x))
Out: array([False,  True])
print(np.logical_not(x))
Out: array([False,  True])

x = [True, False, 0, 1, 3]
print(np.logical_not(x))
Out: array([False,  True,  True, False])
print(np.invert(x))
Out: array([-2, -1, -1, -2, -4])
```

Reply:
replaced